dashboard/app/controllers/reports_controller.rb

```diff
@@ -60,9 +60,23 @@ def search_for_teachers
     authorize! :read, :reports
 
     email_filter = "%#{params[:emailFilter]}%"
-    address_filter = "#{params[:addressFilter]}%"
+    address_filter = "%#{params[:addressFilter]}%"
+
+    # TODO(asher): Determine whether we should be doing an inner join or a left
+    # outer join.
+    @teachers = User.limit(100).where(user_type: 'teacher').where("email LIKE ?", email_filter).where("full_address LIKE ?", address_filter).joins(:followers).group('followers.user_id')
```

Comment:
Given that we're limiting to 100, would it be appropriate to have a notification when that limit is actually being used? Something like "x more results not shown."